src/EventHandlers.ts

```diff
@@ -340,85 +340,85 @@ HolographOperatorContract.FinishedOperatorJob.handler(({ event, context }) => {
       operator: event.params.operator,
       logIndex: event.logIndex,
       eventsSummary: GLOBAL_EVENTS_SUMMARY_KEY,
-    }
+    };
 
-  context.EventsSummary.set(nextSummaryEntity)
+  context.EventsSummary.set(nextSummaryEntity);
   context.HolographOperator_FinishedOperatorJob.set(
     holographOperator_FinishedOperatorJobEntity
-  )
-})
+  );
+});
 
 /// --------------Contract: HolographableContract
 
 // Event: Transfer
 HolographableContract.Transfer.loader(({ event, context }) => {
-  context.EventsSummary.load(GLOBAL_EVENTS_SUMMARY_KEY)
+  context.EventsSummary.load(GLOBAL_EVENTS_SUMMARY_KEY);
 
-  context.Transaction.load(event.transactionHash)
-})
+  context.Transaction.load(event.transactionHash);
+});
 
 HolographableContract.Transfer.handler(({ event, context }) => {
   // create or update transaction
-  handleTransaction(event, context)
+  handleTransaction(event, context);
 
-  const summary = context.EventsSummary.get(GLOBAL_EVENTS_SUMMARY_KEY)
+  const summary = context.EventsSummary.get(GLOBAL_EVENTS_SUMMARY_KEY);
 
   const currentSummaryEntity: EventsSummaryEntity =
-    summary ?? INITIAL_EVENTS_SUMMARY
+    summary ?? INITIAL_EVENTS_SUMMARY;
 
   const nextSummaryEntity = {
     ...currentSummaryEntity,
     holographableContract_TransferCount:
       currentSummaryEntity.holographableContract_TransferCount + BigInt(1),
-  }
+  };
 
   const holographableContract_TransferEntity: HolographableContract_TransferEntity =
     {
       id: event.transactionHash + event.logIndex.toString(),
       _from: event.params._from,
       _to: event.params._to,
-      _tokenId: event.params._tokenId,
+      _tokenId: event.params._value,
```

Comment:
Everything in here is formatting apart from this. Naturally this doesn't quite make logical sense however fixes the ts compiiler error.